tools/release_do_nothing.py

```diff
@@ -508,43 +754,43 @@ def check_rtd(self):
         )
         self.wait_for_done(message)
 
-        message = f"Set {self.git_tag} to Active, un-Hidden."
+        message = f"Set {self.version} to Active, un-Hidden."
         self.wait_for_done(message)
 
-        message = f"Set {self.strings.branch} to Active, Hidden."
+        message = f"Set {self.version.branch} to Active, Hidden."
```

Comment:
New RTD interface: we believe that branches never built before now have to be added using the `Add version` button.